assignments/objects.js

```diff
@@ -18,28 +18,66 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const Kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": function() {
+    return "Hello, my name is" + " " + this.name + "!";
```

Comment:
This could be simplified like this:
```javascript
return "Hello, my name is " + this.name + "!";
```

or, by using a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):
```javascript
return `Hello, my name is ${this.name}!`;
```